DEVELOPMENT.md

```diff
@@ -194,6 +199,9 @@ data:
   clusteringress.class: "istio.ingress.networking.knative.dev"
 ```
 
+You need to set "istio.sidecar.includeOutboundIPRanges" into "*", when you use Minikube or Docker-desktop as
```

Comment:
`*` should be the default setting. I don't think anything should be necessary?

Reply:
Are you suggesting removing this line, or saying keep the default setting for Minikube and Docker-desktop.

I can try "You should keep the default setting "*" for "istio.sidecar.includeOutboundIPRanges", when ...".